### ISSUE TITLE
Players who are not dead are marked dead

### ISSUE THREAD

**Issue by Aasikki** [denverquane/amongusdiscord#53]
I'm having a problem where players who are not dead, are marked dead and can't talk during emergency calls because of this. It happened to me at least once when I was imposter myself and second time for another player. Is there a way I can give more details about this? I had to stop using the bot because of this.

**Comment by denverquane** [comment]
Yeah, if you can provide screenshots of the capture.exe logs of it happens again, in conjunction with the logs.txt that is generated by the discord. exe, that would help a ton :)

**Comment by Aasikki** [comment]
Here's my logs.txt
[logs.txt](https://github.com/denverquane/amongusdiscord/files/5270585/logs.txt)

The first time it happened to me at around 19:39 (player name Sussikki, discord id 121323991221796868)

The second time it happened to my friend at around 20:31 (player name Obama, discord id 230605176648499200)

Both of those times are after is happened as I don't really know how to read the log. I took the times from discord messages that the falsely dead players posted in discord after noticing it.

For now I can't get anything from capture.exe as we already stopped playing for the day, I'll see if my friends are willing to test it out more (they started hating on the bot after we had this issue, which is very unfortunate).
